### ISSUE TITLE
Add a contact form UI 

### ISSUE THREAD

**Issue by shashank-iter** [Web3-Bharat-ITER/UI-Component-Library#32]
It should must include Name, Emaila and message field along with send button. You can make it as creative as you want by adding special CSS effects.

**Comment by riyaku11** [comment]
please assign me thus issue

**Comment by shashank-iter** [comment]
@riyaku11 Sure, Kindly read the contribution guidelines and add a screenshot of your work while making the PR.

**Comment by riyaku11** [comment]
okay @shashank-iter

**Comment by riyaku11** [comment]
done @shashank-iter , please check

**Comment by MangEs23** [comment]
can I still work on this  @shashank-iter ?

**Comment by shashank-iter** [comment]
@MangEs23 Yes you can work on this too, try to be creative. Don't forget to add a screenshot of your work while making the PR.

**Comment by shashank-iter** [comment]
@coder-chahal Yes you can work on it but kindly complete your previous PR and do the changes as told by @Jyo561Python. Just add the files in a folder named as navbar and commit again.

**Comment by coder-chahal** [comment]
@shashank-iter I have done previous PR and add the files in a navbar folder.

**Comment by coder-chahal** [comment]
![img2](https://user-images.githubusercontent.com/110322061/194112566-8b1b2d6b-3f9a-46fa-bd3e-8f554ef60235.png)

**Comment by coder-chahal** [comment]
![img3](https://user-images.githubusercontent.com/110322061/194112618-00d2ccfd-d607-443d-a194-dcbffea2010a.png)

**Comment by 18Ayushi** [comment]
hey! @shashank-iter i want to contribute to this contact form UI. Can u assign this one to me?

**Comment by Jyo561Python** [comment]
Try to submit a unique one

**Comment by 18Ayushi** [comment]
ok! will try my best

**Comment by kasib12** [comment]
@Jyo561Python @shashank-iter Would love to help you all by adding a Contact form styled beautifully with CSS. Assign this issue to me. I'll make sure to read the contribution guidelines.

**Comment by hafidzzakky** [comment]
Hi @shashank-iter, im interest to work with contact form, please assign to me

**Comment by Jyo561Python** [comment]
You have been assigned one issue @hafidzzakky complete it and we will assign you this one😊
